docs/trulens_eval/basic_instrumentation.ipynb

```diff
@@ -1,6 +1,7 @@
 {
```

Comment:
<p>If we're moving the text here from the <code>/api/trucustom</code>  page, should we remove it there?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/truera/trulens/pull/505/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
Good point - probably better to reduce the duplication. I'll reduce what's here, just want to make sure it's mentioned on this page.